packages/contentstack-bulk-publish/src/consumer/publish.js

```diff
@@ -247,7 +247,7 @@ async function performBulkPublish(data, _config, queue) {
       payload['details'] = conf;
       if (bulkPublishObj.apiVersion) {
         if (!isNaN(bulkPublishObj.apiVersion) && bulkPublishObj.apiVersion === apiVersionForNRP) {
-          payload['api_version'] = bulkPublishObj.apiVersion;
+          stack.stackHeaders['api_version'] = bulkPublishObj.apiVersion;
```

Comment:
@sunil-lakshman , previously it's working with payload for NRP. can you pls check whether changes required or not? And also don't think it's good idea to send payload in this format. If it's required then it should be handled from sdk side.
CC: @netrajpatel @shafeeqd959

Reply:
@aman19K Passing api_version in header is removed and set api_version in header handed in SDK side.
CC: @shafeeqd959 @cs-raj